### DISCUSSION TITLE
Both Windows binaries don't start on my PC

### DISCUSSION THREAD

**Discussion by MetinSeven** [visioncortex/vtracer#66] [Q&A] [answered]
Hi,

Thanks for developing Vtracer. I'm eager to try it, but both the 64-bit and 32-bit executables don't work on my Windows 11 PC. I start the .exe and I see a DOS / Shell window (trying to) open for a fraction of a second, then nothing happens.

Any help would be appreciated, thanks in advance.

**Comment by s1291** [comment, marked as answer]
Hi,
Just to clarify, Vtracer is a command-line tool rather than a GUI-based software.

**Comment by MetinSeven** [comment]
Thanks for the clarification. Maybe it's good to emphasize that in the information, as that wasn't clear to me, looking at the GUI screenshots in the info page, which I now realized are from the Web demo.

Will there be a GUI-based release in the near future? I'd love that.

Thanks!

**Reply by s1291** [reply]
I'm not associated with the Vtracer project, so I don't have information about their future releases or plans for a GUI-based version. However, if you're interested in using the web app version on your local machine, you can find instructions here: https://github.com/visioncortex/vtracer/tree/master/webapp

**Reply by mykeehu** [reply]
Looking at the commands, is it possible that the webapp is not for Windows? How can I run it locally on UI on Windows?
